entitlement/src/main/java/org/killbill/billing/entitlement/engine/core/DefaultEventsStream.java

```diff
@@ -239,7 +251,7 @@ public Integer getDefaultBillCycleDayLocal() {
 
     @Override
     public Collection<BlockingState> getBlockingStates() {
```

Comment:
Can we make the code symmetrical with `DefaultSubscriptionBase#getAllTransitions(final boolean includeDeleted)`, i.e the method takes the  `includeDeleted` flag, which is only called from the `BlockingStateOrdering#getBlockingStates()` ?

Reply:
Done